src/UseCases/AddDonation/AddDonationUseCase.php

```diff
@@ -177,4 +179,54 @@ private function getConfirmationMailTemplateArguments( Donation $donation, bool
 		];
 	}
 
+	private function getConfirmationPageArguments( Donation $donation ) {
+		return array_merge( [
+			'donation' => [
+				'id' => $donation->getId(),
+				'status' => $donation->getStatus(),
+				'amount' => $donation->getAmount(),
+				'interval' => $donation->getInterval(),
+				'paymentType' => $donation->getPaymentType(),
+				'optsIntoNewsletter' => $donation->getOptsIntoNewsletter(),
+				'bankTransferCode' => $donation->getBankTransferCode(),
+				// TODO: use locale to determine the date format
+				'creationDate' => ( new \DateTime() )->format( 'd.M.Y' ),
+				// TODO: set cookie duration for "hide banner cookie"
+				'cookieDuration' => ''
+			],
+			$this->getPersonArguments( $donation ),
+			$this->getBankDataArguments( $donation )
+		] );
```

Comment:
This code belongs in the presenter. The fact that there is a template is an implementation detail of the presenter.